core/modules/filter/js/filter.js

```diff
@@ -104,6 +104,33 @@ Backdrop.filterEditorDetach = function(field, format, trigger) {
   }
 };
 
+/**
+ * Provides summary text for the "Formatting options" fieldset, under each
+ * textarea field with a text editor.
+ */
+Backdrop.behaviors.filterFieldsetSummaries = {
+  attach: function (context) {
+    $(context).find('fieldset.filter-wrapper').backdropSetSummary(function (element) {
+      var $element = $(element);
+      var $summary = '';
+      // Look for a select list of text formats.
+      var $select_list = $element.find('select.filterEditors-processed :selected');
+      // Otherwise look for a hidden input element (when the current user has
+      // access to only a single text format).
+      var $input_element = $element.find('input.filterEditors-processed');
```

Comment:
@klonos It works for me. Here's what I meant:

```suggestion
      var $summary = '';
      // Look for a select list of text formats.
      var $select_list = $(element).find('select.filterEditors-processed :selected');
      // Otherwise look for a hidden input element (when the current user has
      // access to only a single text format).
      var $input_element = $(element).find('input.filterEditors-processed');
```